modules/shortcodes/gist.php

```diff
@@ -128,14 +201,13 @@ function github_gist_shortcode( $atts, $content = '' ) {
 
 /**
  * Use script tag to load shortcode in editor.
+ * Can't use wp_enqueue_script here.
  *
  * @since 3.9.0
  *
  * @param string $id The ID of the gist.
- *
- * @return string
  */
 function github_gist_simple_embed( $id ) {
 	$id = str_replace( 'json', 'js', $id );
-	return '<script type="text/javascript" src="https://gist.github.com/' . $id . '"></script>';
+	return '<script src="https://gist.github.com/' . $id . '"></script>'; // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
```

Comment:
Let's escape this so that no future changes on lines above can cause problems.
```suggestion
	return '<script src="' . esc_url( "https://gist.github.com/$id" ) . '"></script>'; // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
```